lib/diagnostic.rb

```diff
@@ -6,21 +6,22 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'ruby lib/diagnostic.rb'
+
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'irb'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'best_movie = "star wars episode viii"'
```

Comment:
We want you to name a variable after "star wars episode viii" using Ruby's naming convention (snake case).  `star_wars_episode_viii`.